src/core/config/channel_monitor.h

```diff
@@ -35,6 +35,16 @@
 #ifndef CONFIG_CHANNEL_MONITOR_H_
 #define CONFIG_CHANNEL_MONITOR_H_
 
+/**
+ * @addtogroup config-channel-monitor
+ *
+ * @brief
+ *   This module include configuration variables for Channel Monitor.
```

Comment:
```suggestion
 *   This module includes configuration variables for Channel Monitor.
```

Can you double-check all the files in the PR?  Many of them say "include" instead of "includes", they should all be updated.